panel/models/vtk.py

```diff
@@ -0,0 +1,28 @@
+# coding: utf-8
+"""
+Defines custom VtkPlot bokeh model to render Vtk objects.
+"""
+import os
+
+from bokeh.core.properties import String, Bool, Override
+from bokeh.models import HTMLBox
+
+from ..util import CUSTOM_MODELS
+
+
+class VtkPlot(HTMLBox):
```

Comment:
Should this be VTKPlot?  [VTK](https://vtk.org/) is always capitalized when I see it.

Reply:
+1